data_fixes/MC104.13/remove_digital_objects_MC104.rb

```diff
@@ -0,0 +1,36 @@
+require 'archivesspace/client'
+require 'active_support/all'
+require 'nokogiri'
+require_relative '../../../helper_methods.rb'
+
+
+@client = aspace_staging_login
+log = "delete_dos_MC10413.txt"
+
+start_time = "Process started: #{Time.now}"
+puts start_time
+
+resource = @client.get("/repositories/3/resources/1862/ordered_records").parsed
+#greedy '+'' matches second 's' as intended
+ids = resource['uris'].map {|uri| uri['ref'].gsub!(/^.+s\//, "") unless uri['level'] == 'collection'}
+aos =
```

Comment:
I don't think this variable ever gets used, so the assignment on this line is unnecessary

Reply:
Removed, thanks!